frontend/src/components/company-survey-v2/grid-connection.tsx

```diff
@@ -9,6 +9,7 @@ import {Storage} from './storage'
 import {Supply} from './supply'
 import {Transport} from './transport/transport'
 import {GridExpansion} from "./grid-expansion";
+import {PandenSelect} from "./panden-select"
```

Comment:
The code is a bit Dutch and English. Should we try to keep the code more english and the text in the UI in Dutch.

Reply:
I use pand and buurtcode because both are domain terms. I don't think domain terms should be translated.

In our model "pand" is used for the location(s) of a company and "building" for the heat characteristics.